src/content/1/en/part1d.md

```diff
@@ -290,7 +290,7 @@ const App = (props) => {
 }
 ```
 
-Now the behavior of the component depends on whether or not any buttons have been clicked. If not, meaning that the <em>allClicks</em> array is empty, the component renders a div component with some instructions:
+Now the behavior of the component depends on whether or not any buttons have been clicked. If not, meaning that the <em>allClicks</em> array is empty, the component renders a div element with some instructions instead:
```

Comment:
it's actually a component so not sure about this change, but i feel not repeating "component" helps make things easier to understand